src/rad/inspector/wxfbadvprops.cpp

```diff
@@ -578,10 +594,33 @@ wxVariant wxFBBitmapProperty::ChildChanged(wxVariant& thisValue, const int child
 
                     break;
                 }
-                // 'Load From Art Provider'
-                case 4:
+				// 'Load From XRC'
+				case 4:
+				{
+					if (prevSrc != 4)
+					{
+						for (unsigned int i = 1; i<count; i++)
+						{
+							wxPGProperty *p = Item(i);
```

Comment:
Move this into the `if( )`

Reply:
This raises the urgend need inside me to adjust the other cases as well, after all they are all copy-adjust-pasted and it just doesn't look right if i only change this one but not the other ones...